src/npc.h

```diff
@@ -5,6 +5,7 @@
 #include "player.h"
 #include "faction.h"
 #include "pimpl.h"
+#include "overmap_location.h"
```

Comment:
You don't need the definition of `overmap_location` to use a `int_id<overmap_location>`, a forward declaration of `overmap_location` is enough.